config/locales/de.yml

```diff
@@ -0,0 +1,204 @@
+de:
+  title: "RubyGems.org"
+  subtitle: "Deine Community des Gem-Hostingservices"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "Zugriff verweigert. Bitte melden Sie in https://rubygems.org mit einem Konto an"
+  feed_subscribed: "RubyGems.org | Abonnierte Gems"
+  feed_latest: "RubyGems.org | Neueste Gems"
+  time_ago: "seit %{duration}"
+  form_disable_with: "Bitte warten..."
+  update: Aktualisieren
+  edit: Bearbeiten
+  sign_in: "Anmelden"
+  sign_up: "Registrieren"
+  locale_name: "German"
+  this_rubygem_could_not_be_found: "Dieser Ruby Gem konnte nicht gefunden werden."
+
+  dashboards:
+    show:
+      latest: "Letzte Aktualisierungen"
+      latest_title: "Neueste Aktualisierungen RSS Feed"
+      no_subscriptions: "Du hast noch keine Gems abonniert. Besuche %{gem_link} um ein Gem zu abonnieren!"
+      gem_link_text: "Gem-Seite"
+      mine: "Meine Gems"
+      no_owned: "Du hast noch keine Gems gepusht. Schau doch vielleicht die Dokumentation auf %{creating_link} an und %{migrating_link} ein Gem from RubyForge."
+      creating_link_text: 'Erstellen'
+      migrating_link_text: 'Migrieren'
+      my_subscriptions: 'Abonnements'
+
+  home:
+    index:
+      downloads_counting: 'Gezählte Downloads'
+      find_blurb: "Finde, installiere und veröffentliche RubyGems."
+      learn:
+        install_rubygems: "Installiere RubyGems"
+
+  layouts:
+    application:
+      header:
+        dashboard: "Dashboard"
+        sign_in: "Anmelden"
+        sign_up: "Registrieren"
+        sign_out: "Abmelden"
+        search_gem: "Suche Gems&hellip;"
+      footer:
+        status: "Status"
+        uptime: "Betriebszeit"
+        source_code: "Code"
+        data: "Daten"
+        discussion_forum: "Diskussion"
+        statistics: "Statistiken"
+        blog: "Blog"
+        about: "Über uns"
+        help: "Hilfe"
+        guides: "Dokumentation"
+        contribute: "Beitragen"
+        supported_by: "Unterstützt von"
+        hosted_by: "Gehostet von"
+        designed_by: "Design von"
+        resolved_with: "Aufgelöst mit"
+        optimized_by: "Optimiert mit"
+        tracking_by: "Getrackt von"
+        monitored_by: "Überwacht von"
+        gems_served_by: "Gems serviert von"
+
+  pages:
+    about:
+      title: Über uns
+      purpose:
+        header: "Willkommen in %{site}, die Ruby-Community für Gem-Hostingservice. Die Gründe sind vielfältig:"
```

Comment:
`s/in/auf/`. And "Ruby-Community für Gem-Hostingservice" makes no sense. What are you trying to say?